src/table/table-role/grid-navigation.tsx

```diff
@@ -100,13 +91,28 @@ class GridNavigationHelper {
     this._pageSize = pageSize;
   }
 
+  public refresh() {
+    if (this._table) {
```

Comment:
Can this be run before the processor has initialised?

Reply:
It should not, but that depends on the order of effects execution. If the effect that fires the update runs before the one that initialises the table there going to be an error.